### ISSUE TITLE
question about the process to use your project

### ISSUE THREAD

**Issue by ChasingStar95** [MarcCote/iRBM#5]
Thank you for your listen! I don't know how to run you project correctly. Can you give me some instructions 
about how to run the project correctly. For example, when I try to run train_model.py,  an error comes at with the warning that 
usage: train_model.py [-h] (--nb-epochs N | --max-epoch N) [--batch-size M]
                      [--dataset-percent X]
                      (--ConstantLearningRate LR | --ADAGRAD LR [EPS=1e-6])
                      [--L1Regularization LAMBDA | --L2Regularization LAMBDA]
                      [--name NAME] [--seed SEED] [--keep K] [-f]
                      ...
train_model.py: error: too few arguments 
    Can you give some instructions. Thank you very much.

**Comment by MarcCote** [comment]
See https://github.com/MarcCote/iRBM/blob/master/README.md but let me know if it misses anything. Thanks.

**Comment by ChasingStar95** [comment]
Thank you for your advice. The main problem is that I don't know where to modify parameters which you advise. Maybe I have ignored some special information in train_model.py and I haven't change anything in the whole project. Can you just show me some pictures about codes that I have to modify or instructions about which code section in train_model.py that I must to modify.

**Comment by MarcCote** [comment]
Sorry, for the (very) late reply. There is no code to change, only the arguments when calling the training script. The README contains the actual commands you need to run in order to reproduce the results in the paper.

I'm closing this issue but feel free to re-open it if needed.
